conozco.py

```diff
@@ -615,39 +615,39 @@ def pantallaStats(self):
                         int(100*scale+shift_y)),
                         COLOR_ACT_NAME)
         msg = _('Total score: %s') % self._score
-        self.mostrarTexto(unicode(msg, 'UTF-8'),
+        self.mostrarTexto(str(msg, 'UTF-8'),
```

Comment:
Why is it necessary to convert the translated string in line 617 to another string?  Also occurs in subsequent lines.  If you're not sure what type it is, use Pdb.

Reply:
i generally used 2to3 as it said in the python porting guide, i'm happy to revert these changes back.

Reply:
thanks for asking.  yes, the porting guide said to use 2to3, then it said to _review every change made_.  which means to look at the change, figure out if the change is appropriate, and that the lines of code serve a purpose.  i'm not sure these lines serve any purpose in python 3, though they used to in python 2.  that's why i asked.  i don't think reverting them back solves the question of if they are a correct change.